docs/README.md

```diff
@@ -24,26 +24,65 @@ The .md files in the `docs/` directory are the docs. See the [Docusaurus website
 - [This Github Action](../.github/workflows/release-please.yml) is Release-Please, a framework made to organize different commits into releases. When it merges to master, it runs. When it runs, it builds the dependencies and cuts a new version of the docs, with the same tag that is being released
 
-The `#include_aztec_version` and `#include_code` macros look for the version tag in an environment variable `COMMIT_TAG`, so you can build the docs specifying a version with the following command (e.g. for v0.84.0). Remove the versions listed in `versions.json` before running:
+The `#include_aztec_version` and `#include_code` macros look for the version tag in an environment variable `COMMIT_TAG`, so you can build the docs specifying a version with the following command (e.g. for v0.86.0). Remove the versions listed in `versions.json` before running:
 
 ```bash
-COMMIT_TAG=v0.84.0 yarn build
+COMMIT_TAG=v0.86.0 yarn build
 ```
 
-You can add the aztec version to a docs page without the `v` prefix with `#include_version_without_prefix`, so COMMIT_TAG `v0.85.0` will render as `0.85.0`.
+You can add the aztec version to a docs page without the `v` prefix with `#include_version_without_prefix`, so COMMIT_TAG `v0.86.0` will render as `0.86.0`.
 
 ### How do I change the versions that show in the website
 
 When docusaurus builds, it looks for the `versions.json` file, and builds the versions in there, together with the version in `docs`.
 
+## Adding A Version
+
+Versions of docs can be created automatically via a github action, or manually.
+
+From earlier, the release github workflow is triggered on pushes to `master` or `next` branches and does two things:
+1. using google's `Release-Please` action to create a new PR and version tag
+2. creates a build artifact for the new version of the docs (a folder in `versioned_docs`)
+
+To manually perform step two locally:
+- Checkout the branch/tag of the version of docs you'd like to create
+- From the desired branch/tag, perform the "cut version" step of the github action ([Release-Please.yml](https://github.com/AztecProtocol/aztec-packages/blob/next/.github%2Fworkflows%2Frelease-please.yml#L72-L78)). Note: you will have to set COMMIT_TAG to "v0.xx.x"
+- Then the newly created version of docs under `docs/versioned_docs` can be stashed, to later pop on your branch that you want to commit the new versioned docs to.
+- Since the new version is likely taking the place of latest:
+  - Go to `versioned_docs` and rename `version-latest` to it's version (eg `version-v0.xx.x`)
+  - Rename the newly created version as `version-latest`
+- If desired, delete any past versions we no longer want to display
+- Update `versions.json`
+
+### A special note about branches
+
+Adding docs:
+- All docs changes, especially changes to `versioned_docs`, should only go to the `master` branch
+- Exceptionally when documenting new protocol changes, these should go to the `next` branch, within `docs/docs`
```

Comment:
I think all docs changes should go into master. We don't have to publish what's in docs/docs (like we do now), which will contain future changes. This should prevent merge conflicts, otherwise I don't see how to prevent conflicts when merging master into next.

Reply:
Updated.